perfdash/google-gcs-downloader.go

```diff
@@ -22,65 +22,49 @@ import (
 	"os"
 
 	"k8s.io/contrib/test-utils/utils"
-	"k8s.io/kubernetes/pkg/util/sets"
 )
 
 // constants to use for downloading data.
 const (
-	jobName = "kubernetes-e2e-gce-scalability"
 	logFile = "build-log.txt"
 )
 
 // GoogleGCSDownloader that gets data about Google results from the GCS repository
 type GoogleGCSDownloader struct {
-	startFrom            int
+	Builds               int
```

Comment:
The reason for previous logic was to make it easier to make not backward-compatible changes in e.g. format of data. After doing that one could just set a build number to the one when given change went live.

I liked old behavior, but if you have an argument (except simplicity) why new one is better then I'm OK with this change.